pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -411,3 +416,25 @@ func validateCloneVolumeSnapshotSupportVMSnapshotContent(snapshotContents *v1alp
 
 	return result
 }
+
+func validateNewMacAddresses(vmClone *clonev1alpha1.VirtualMachineClone) []metav1.StatusCause {
+	var causes []metav1.StatusCause
+
+	if vmClone.Spec.NewMacAddresses == nil {
+		return nil
+	}
+
+	regex := regexp.MustCompile(`^([0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2}$`)
+	for ifaceName, mac := range vmClone.Spec.NewMacAddresses {
+		if !regex.MatchString(mac) {
+			causes = append(causes, metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueInvalid,
+				Message: "MAC address is not valid",
+				Field:   k8sfield.NewPath("spec").Child("newMacAddresses").Child(ifaceName).String(),
+			})
+		}
+
+	}
+
+	return causes
```

Comment:
Please use `net.ParseMAC(mac)` instead of a custom regex.
I would suggest you to take inspiration from https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go#L399-L418

Reply:
Hi, Thanks a lot for your timely reply. I have already finished it.